netty/src/main/java/io/grpc/transport/netty/Utils.java

```diff
@@ -177,13 +173,8 @@ private static Http2Headers convertMetadata(Metadata headers) {
 
     @Override
     public EventLoopGroup create() {
-      return new NioEventLoopGroup(0, new ExecutorServiceFactory() {
```

Comment:
Doesn't this constructor give us basically the same functionality?

https://github.com/netty/netty/blob/4.1/transport/src/main/java/io/netty/channel/nio/NioEventLoopGroup.java#L53-L59